setup.cfg

```diff
@@ -27,6 +27,8 @@ install_requires =
     numpy
     lmoments3
```

Comment:
Just a note - I had an error pop up when I used the lmoments3 that loads with climakitae when I called threshold_tools.py functions.  I directly pip installed from a git and it worked.  May be a source of error down the road.

Reply:
Good to know... this is not a part of this PR though -- but I'm glad you tried testing those things in concert.